warmup/schemas/mysql/mysql.sql

```diff
@@ -29,6 +29,16 @@ CREATE TABLE IF NOT EXISTS `config_search` (
   CONSTRAINT `cs_id_id` FOREIGN KEY (`_id`) REFERENCES `config` (`_id`) ON DELETE CASCADE ON UPDATE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE IF NOT EXISTS `config_metadata` (
+  `_id` varchar(32) NOT NULL,
+  `name` varchar(64) NOT NULL,
+  `value` text NOT NULL,
+  KEY `value` (`value`(255)),
+  KEY `name` (`name`),
+  KEY (`_id`),
+  CONSTRAINT `cm_id_id` FOREIGN KEY (`_id`) REFERENCES `config` (`_id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
UNIQUE constraint could go here. Basically in a fairly new (April 1st) Install, I had duplicate values stored which were from the application when running the `INSERT INTO SELECT FROM WHERE` part of the queries